### DISCUSSION TITLE
Which version is compatible with Wildfly 26

### DISCUSSION THREAD

**Discussion by omasseau** [wildfly/wildfly-arquillian#467] [Q&A]
Hello,

I cannot find the infos on which version I should use with Wildfly 26.
In the REAME it says : "Versions under 3.0.0 should work with any version of WildFly and JBoss EAP 7.0 and higher"

But we get this error with Wildfly 26 : 
`java.lang.ClassNotFoundException: org.jboss.threads.AsyncFuture`

Should I use the 4.0.1 version or the 5.0.1 version ?

**Comment by jamezp** [comment]
For WildFly 26 I would suggest 4.0.1. Version 5.x is for Jakarta EE 10, so WildFly 27+.

The 3.0 version should work, but note there will be no more bug fixes for that release stream. I've seen that issue before and it's just a dependency issue. However, I think it's fixed in 4.0. I couldn't find a JIRA for it, but I'm pretty sure it's been fixed :)
